scope & closures/apC.md

```diff
@@ -107,8 +108,8 @@ var obj = {
 obj.cool(); // more awesome
 ```
 
-Whether you prefer the new lexical-this behavior of arrow-functions, or you prefer the tried-and-true `bind()`, it's important to note that arrow-functions are **not** just about less typing of "function".
+Independentemente se você prefere o novo comportamento `lexical-this` das arrow-functions, ou se você prefere o testado e comprovado `bind()`, é importante notar que `arrow-functions` **não** são apenas para economizar caracteres ao digitar "function".
 
-They have an *intentional behavioral difference* that we should learn and understand, and if we so choose, leverage.
+Elas têm uma *diferença de comportamento intencional* que devemos entender e aprender.
```

Comment:
É um jargão em inglês (provavelmente vindo da bíblia). *and if we so choose* é como se ele dissesse *ou se preferir*

No contexto:

elas têm uma *diferença de comportamento intencional* (ou se preferir: importância diferente) que devemos entender e aprender.